client/battle/CBattleInterface.cpp

```diff
@@ -104,7 +106,7 @@ CBattleInterface::CBattleInterface(const CCreatureSet *army1, const CCreatureSet
 	currentlyHoveredHex(-1), attackingHex(-1), stackCanCastSpell(false), creatureCasting(false), spellDestSelectMode(false), spellToCast(nullptr), sp(nullptr),
 	creatureSpellToCast(-1),
 	siegeH(nullptr), attackerInt(att), defenderInt(defen), curInt(att), animIDhelper(0),
-	myTurn(false), moveStarted(false), moveSoundHander(-1), bresult(nullptr)
+	myTurn(false), moveStarted(false), moveSoundHander(-1), bresult(nullptr), battleActionsStarted(false)

```

Comment:
I think in class initialization will be better one

Reply:
It probably would, but what about consistency with other variables atm? Some initialized here, some in-class would result in a mess.

Reply:
I thought that constructor is a mess anyway, keeping consistency makes no sense when itself it's broken, but you can do as you wish